packages/typescript/template/package.json

```diff
@@ -30,19 +30,19 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^4.5.0",
-    "@enact/i18n": "^4.5.0",
-    "@enact/sandstone": "^2.5.0",
-    "@enact/spotlight": "^4.5.0",
-    "@enact/ui": "^4.5.0",
-    "@types/node": "^12.0.0",
-    "@types/react": "^18.0.0",
-    "@types/react-dom": "^18.0.0",
-    "ilib": "^14.14.0",
+    "@enact/core": "^4.5.2",
+    "@enact/i18n": "^4.5.2",
+    "@enact/sandstone": "^2.5.5",
+    "@enact/spotlight": "^4.5.2",
+    "@enact/ui": "^4.5.2",
+    "@types/node": "^18.11.0",
```

Comment:
Note for the next reviewer: Since we were already using typescript 4.6(with this change 4.8), I guess this change is necesary